src/pages/workspaces/workspace/Analyses.js

```diff
@@ -50,36 +49,30 @@ import ExportAnalysisModal from 'src/pages/workspaces/workspace/notebooks/Export
 import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
 
 
-const analysisCardCommonStyles = listView => _.merge({ display: 'flex' },
-  listView ?
-    {
-      marginBottom: '0.5rem',
-      flexDirection: 'row',
-      alignItems: 'center'
-    } :
-    {
-      margin: '0 2.5rem 2.5rem 0',
-      height: 100,
-      width: 400,
-      flexDirection: 'column',
-      padding: 0
-    }
-)
-
 const noWrite = 'You do not have access to modify this workspace.'
 
 const sortTokens = {
-  lowerCaseName: notebook => notebook.name.toLowerCase()
+  name: notebook => notebook.name.toLowerCase()
 }
-const defaultSort = { label: 'Most Recently Updated', value: { field: 'updated', direction: 'desc' } }
-const sortOptions = [
-  defaultSort,
-  { label: 'Least Recently Updated', value: { field: 'updated', direction: 'asc' } },
-  { label: 'Alphabetical', value: { field: 'lowerCaseName', direction: 'asc' } },
-  { label: 'Reverse Alphabetical', value: { field: 'lowerCaseName', direction: 'desc' } }
-]
+const defaultSort = { label: 'Most Recently Updated', value: { field: 'lastModified', direction: 'desc' } }
+
+const analysisContextMenuSize = 18
+
+const AnalysisCardHeaders = Utils.memoWithName('AnalysisCardHeaders', ({ sort, onSort }) => {
+  return div({ style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem 0 1.5rem', marginBottom: '0.5rem' } }, [
+    div({ 'aria-sort': ariaSort(sort, 'Application'), style: { flex: 1 } }, [
+      h(HeaderRenderer, { sort, onSort, name: 'application' })
+    ]),
+    div({ 'aria-sort': ariaSort(sort, 'name'), style: { flex: 5 } }, [
+      h(HeaderRenderer, { sort, onSort, name: 'name' })
+    ]),
+    div({ 'aria-sort': ariaSort(sort, 'lastModified'), style: { flex: '0 0 10%', justifyContent: 'flex-left', display: 'flex' } }, [
```

Comment:
```suggestion
div({ 'aria-sort': ariaSort(sort, 'lastModified') }, [
```

This header wrapped when the window size changed, I think this solves the issue

Reply:
hmmm... this was a massive pain to sort out to between left-aligning the content within the div, prevent it from shrinking and hugging the clickables in the right of the flex box, and ensuring the column and headers line up. I ended up copying the code in the  table in the billing tab, which is what UX wants to use as the source for table styling. It should fix the issue you mention and all the things listed above, but with different code.